custom-elements-manifest.config.js

```diff
@@ -14,22 +14,12 @@ import { moduleFileExtensionsPlugin } from 'cem-plugin-module-file-extensions';
 import defineElementPlugin from './scripts/define-element-plugin.js';
 
 export default {
-    globs: [
-        '**/sp-*.ts',
-        '**/overlay-trigger.ts',
-        '**/src/[A-Z]*.ts',
-        '**/src/elements/[A-Z]*.ts',
-        '**/tools/shared/src/*.ts',
-    ],
```

Comment:
@castastrophe removing this breaks the define-element-plugin script which is used in build:cem. Can you share why these were removed?

Reply:
read the config docs. i think if we want to remove the globs, we need to remove the analyze command from storybook-decorator. I don't believe we ship that in a way that we would need a cem for it. please correct me if I'm wrong.

Reply:
I figured out the issue was the analyze command glob pattern in story-decorator. 

a full custom-element.json is generated and stored in documentation project. story-decorator is shipped as an npm package like our components.

Reply:
pushing up a fix